docs/building-and-deploying-the-sample-solution.md

```diff
@@ -102,7 +100,7 @@ The following sections describe these steps in more detail.
 
 ### Verifying the Azure Storage Configuration
 
-The provisioning script uses Azure blob storage to store data after it has been captured by the hot path for further analysis. The cold path also writes data to blob storage. Furthermore, the warm path retrieves reference data used for generating reporting data from blob storage. Perform the following steps to verify the configuration Azure storage:
+The provisioning script generates Azure blob storage to the raw event store data after it has been captured by Stream Analytics for further analysis. A separate Stream Analytics job retrieves reference data used for generating reporting data from blob storage. Perform the following steps to verify the configuration Azure storage:
```

Comment:
> A separate Stream Analytics job retrieves reference data used for generating reporting data from blob storage. 

I think that "utilizies" or something similiar with be better than "retrieves" here.